airbyte-integrations/connectors/source-amazon-seller-partner/acceptance-test-config.yml

```diff
@@ -1,100 +1,127 @@
 connector_image: airbyte/source-amazon-seller-partner:dev
-tests:
+test_strictness_level: high
```

Comment:
I don't think it is fair to enable a high test strictness level if so many streams are declared empty in the basic read. @YowanR could you evaluate if we could make seeding efforts to have more test data and less "empty streams"?

Reply:
@davydov-d I would suggest working on the `high` test strictness level from another PR so that your other changes are not blocked by my request.

Reply:
I'm not working on this PR, @lazebnyi is :)

Reply:
This is also not a GA connector. Agreed with @alafanechere. Let's not enable high strictness test here and do it in another PR. @lazebnyi please create another issue for this and we'll have the Unblockers look at adding more data